backend/tournesol/migrations/0015_auto_20211221_1114.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.10 on 2021-12-21 11:14
```

Comment:
I ususally try to give more explicit names to automatically generated migrations (it makes it slightly easier to work with afterwards).